eth/handler.go

```diff
@@ -851,9 +851,6 @@ func (pm *ProtocolManager) handleMsg(p *peer) error {
 		if err := msg.Decode(&vote); err != nil {
 			return errResp(ErrDecode, "msg %v: %v", msg, err)
 		}
-		// Mark the peer as owning the vote and process it
-		// because peer has 2 address sender and receive, so use p.id to find the right address
-		p = pm.peers.Peer(p.id)
```

Comment:
what's the reason for removing this?

Reply:
This is added by this [PR](https://github.com/hash-laboratories-au/XDPoSChain/pull/68) , to avoid duplicate messages receiving from 1 node and send back to same node. This is behaviour is from original code. And now we have crash issue after adding this code, so revert it back. It may still have duplicate message issue but that can be addressed later or ignore it.